definitions/divorce/json/CaseEventToFields/CaseEventToFields-prod.json

```diff
@@ -1 +1,186 @@
-[]
+[
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
if you add anything to any "prod" file it means that as soon as you merge it this should be released.

Are you sure this is what you want?

Reply:
ok, it's fine. My mistake. It was moved from the core file.

it will not change prod xlsx.

Reply:
Yes, these are the existing items.
I must make sure that my changes are behind a feature toggle. I remove them from the regular "CaseEventToFields.json" file and put the existing code in -prod and the updated one in its own nopnprod file